pages/doc/csp_invite-AoA-users_tutorial.md

```diff
@@ -52,9 +57,10 @@ VMware Cloud uses organizations to provide controlled access to one or more serv
 1. Click your username and click **My Account**.
 2. On the **My Roles** tab you can see what organization roles are assigned to you.
 
-If do not have the VMware Cloud **Organization Owner** or **Organization Administrator** role assigned, you need to request them. To understand who the VMware Cloud **Organization Owner** or **Organization Administrator** users are, you can chat with VMware Support or file a VMware Cloud services support request. See [How do I get support](https://docs.vmware.com/en/VMware-Cloud-services/services/Using-VMware-Cloud-Services/GUID-E4DC731F-C039-4FB2-949E-9A61584CD5BF.html) in the VMware Cloud services product documentation.
+If do not have the VMware Cloud **Organization Owner** or **Organization Administrator** role assigned, you need to request them. To understand who the VMware Cloud **Organization Owner** or **Organization Administrator** users are, you can chat with our Technical Support team or file a VMware Cloud services support request. See [How do I get support](https://docs.vmware.com/en/VMware-Cloud-services/services/Using-VMware-Cloud-Services/GUID-E4DC731F-C039-4FB2-949E-9A61584CD5BF.html) in the VMware Cloud services product documentation.
```

Comment:
If do not have the VMware Cloud **Organization Owner** or **Organization Administrator** role assigned, you need to request them.  -----> 

If you do not have the VMware Cloud **Organization Owner** or **Organization Administrator** role assigned, you need to talk to your Administrator. 

How can they request it? Is that what the second sentence is for? :)

Reply:
We don't have a definition for Administrator. I think the second sentence is for that (Margarita should have consulted with the team).

Reply:
There is a typo here.. If do not have the VMware  --> If you do not have the VMware